package.json

```diff
@@ -40,31 +40,30 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^4.21.0"
+    "@typescript-eslint/experimental-utils": "^4.24.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^12.1.1",
-    "@commitlint/config-conventional": "^12.1.1",
-    "@types/jest": "^26.0.22",
-    "@typescript-eslint/eslint-plugin": "^4.21.0",
-    "@typescript-eslint/parser": "^4.21.0",
+    "@commitlint/cli": "^12.1.4",
+    "@commitlint/config-conventional": "^12.1.4",
+    "@types/jest": "^26.0.23",
+    "@typescript-eslint/eslint-plugin": "^4.24.0",
+    "@typescript-eslint/parser": "^4.24.0",
     "cpy-cli": "^3.1.1",
-    "eslint": "^7.26.0",
+    "eslint": "^7.27.0",
     "eslint-config-kentcdodds": "^18.1.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.23.2",
+    "eslint-plugin-import": "^2.23.3",
```

Comment:
This changes the order of the imports again. I'm really thinking about fixing exact dependencies versions in package.json to avoid unexpected behaviors as we got recently.

Reply:
It was a bug in the plugin, which is now fixed.

The current import order is how it should have been the whole time

Reply:
Yep, but that made other CI runs fail because they installed the latest version, ending in a different formating. Hence the idea of fixing exact versions in package.json

Reply:
This is kind of a special case as `v2.23.0` made a breaking change, which shouldn't have happened and is now fixed in `v2.23.3`

Reply:
But it can happen (especially with ESLint and its plugins), and happened a couple of times in last month. So we should probably fix ESLint and its plugins at least, for the same reason prettier is fixed to exact version.

Reply:
I never had problems in the past with it, except now with `eslint-plugin-import` tbh, so not sure if we should really do that tbh.

We get a lot of benefits from not using exact version numbers imo

Reply:
What are the benefits tho? It's doing more harm than good lately.